types/types.go

```diff
@@ -58,7 +58,7 @@ type Image interface {
 	IntendedDockerReference() string
 	// Manifest is like ImageSource.GetManifest, but the result is cached; it is OK to call this however often you need.
 	// NOTE: It is essential for signature verification that Manifest returns the manifest from which LayerDigests is computed.
-	Manifest() ([]byte, error)
+	Manifest() ([]byte, string, error)
```

Comment:
That ends up `"unsupported manifest media type "`, with a trailing space.  The trailing space kinda is generic enough, but the `unsupported media type` is not necessarily true—the type just couldn’t be guessed. (At the moment there may be no difference because all our manifest types are self-describing. Is that likely to remain true?)